src/01-functions.js

```diff
@@ -24,8 +26,12 @@ function alwaysTrue() {}
  *  greet("Ian");
  *  //> "Hello, Ian."
  */
-function greet() {}
+function greet(name) {
+  name.charAt(0).toUpperCase;
+  return "Hello, " + `${name}` + "."; 
```

Comment:
When using template strings we normaly do not use the ` + ` operator to concatenate - template strings are usually used a replacement.  We can write ` Hello, ${name}.`